source/distributed/tria.cc

```diff
@@ -1689,34 +2224,45 @@ namespace parallel
 
       if (this->my_subdomain == 0)
         {
-          const unsigned int buffer_size_per_cell =
-            (cell_attached_data.cumulative_fixed_data_size > 0 ?
-               cell_attached_data.cumulative_fixed_data_size +
-                 sizeof(CellStatus) :
-               0);
-
           std::string   fname = std::string(filename) + ".info";
           std::ofstream f(fname.c_str());
-          f << "version nproc attached_bytes n_attached_objs n_coarse_cells"
-            << std::endl
-            << 2 << " "
+          f << "version nproc n_attached_objs n_coarse_cells" << std::endl
+            << 3 << " "
             << Utilities::MPI::n_mpi_processes(this->mpi_communicator) << " "
-            << buffer_size_per_cell << " "
             << cell_attached_data.pack_callbacks.size() << " "
             << this->n_cells(0) << std::endl;
         }
 
-      if (cell_attached_data.cumulative_fixed_data_size > 0)
+      // each cell should have been flagged `CELL_PERSIST`
+      for (auto &quad_cell_rel : local_quadrant_cell_relations)
```

Comment:
The variable is unused in `Release` mode so we need a `(void) quad_cell_rel` below. While being here, you can also make the variable `const`.